code/modules/reagents/Chemistry-Recipes.dm

```diff
@@ -95,7 +95,7 @@
 	
 /datum/chemical_reaction/explosion_potassium/holy/on_reaction(var/datum/reagents/holder, var/created_volume)
 	..()
-	playsound('sound/misc/holyhandgrenade.ogg', 100, 1)
+	playsound(src, 'sound/misc/holyhandgrenade.ogg', 100, 1)
```

Comment:
maybe choosing the holder as source is better?

Reply:
I literally added that line originally on kurfursten's suggestion so I don't know.

Reply:
jknpj is right, use the holder

Reply:
>jknpj is right, use the holder
how though

Reply:
instead of `src`, use `holder.my_atom`

Reply:
Ok, on it.